src/routes/docs/tutorials/nuxt-ssr-auth/step-3/+page.markdoc

```diff
@@ -25,7 +25,6 @@ export const SESSION_COOKIE = "my-custom-session";
 
 export function createAdminClient() {
   const client = new Client()
-    .setEndpoint(process.env.APPWRITE_ENDPOINT!)
```

Comment:
Other parts of the tutorial needs to be updated too since this is removed, right?